# Message for Panda robot joint positions (7 DOF)
float64[7] joints
